src/apps/relay/prom_server.c

```diff
@@ -41,7 +45,15 @@ void start_prometheus_server(void) {
     nlabels++;
   }
 
-  // Create traffic counter metrics
+  // Create STUN counters
+  stun_binding_request = prom_collector_registry_must_register_metric(
+      prom_counter_new("turn_traffic_rcvp", "Incoming STUN Binding requests", 0, NULL));
+  stun_binding_response = prom_collector_registry_must_register_metric(
+      prom_counter_new("turn_traffic_rcvp", "Outgoing STUN Binding responses", 0, NULL));
```

Comment:
shouldn't it be _sentp instead of _rcvp if it is for "Outgoing"

Reply:
Names are completely wrong :)

Reply:
Renamed